tensorboard/plugins/profile/profile_plugin.py

```diff
@@ -158,7 +159,7 @@ def data_impl(self, run, tool):
       return None
     if tool == 'trace_viewer':
       return process_raw_trace(raw_data)
-    if tool == 'op_profile' or tool == 'input_pipeline_analyzer':
+    if tool in set(['op_profile', 'input_pipeline_analyzer', 'overview_page']):
```

Comment:
Could we maybe store this set as a frozenset at the top of the file? ie,

_RAW_DATA_TOOLS = frozenset(
    ['input_pipeline_analyzer', 'op_profile', 'overview_page'])

Reply:
Fixed.